fuel-relayer/README.md

```diff
@@ -3,14 +3,8 @@
 
 Functionalities expected from Relayer:
 
-* Validator staking (Handle ValidatorList and use it for fuel-bft):
```

Comment:
Why are so much of the original `fuel-relayer` docs being removed? Would be nice to have them updated where possible. I see a lot of changes also removed a lot of the related `eth` stuff, so having somewhere documenting the current rationale of the `relayer` design and why its currently structured this way would be helpful